packages/kbn-test/src/es/test_es_cluster.ts

```diff
@@ -77,7 +77,7 @@ export interface CreateTestEsClusterOptions {
    * you'll likely need to use `basic` or `gold` to prevent the test from failing
    * when the license expires.
    */
-  license?: 'basic' | 'gold' | 'trial'; // | 'oss'
+  license?: ArtifactLicense;
```

Comment:
@delanni license type mismatching was quite interesting finding. I would like Ops team to share thoughts on artefact license vs product license as this change makes 'gold' not acceptable.

```
export type ArtifactLicense = 'oss' | 'basic' | 'trial';
```

Reply:
I also found this, but I don't know much about the context here, or what "gold" is supposed to refer to :)

Reply:
Original question was if we need to add 'gold', but this type is not matching product license type anyways. 

https://github.com/elastic/kibana/blob/b96dfb698ae3c2ccf05b126bec79c4e57f2b86b2/x-pack/plugins/licensing/common/types.ts#L10-L17

@elastic/kibana-operations to confirm we are good with what it defines at the moment.